events_mgmt/src/main/java/com/adobe/aio/event/management/model/RegistrationUpdateModel.java

```diff
@@ -58,11 +61,16 @@ public class RegistrationUpdateModel {
     this.webhookUrl = webhookUrl;
     this.eventsOfInterestInputModels = eventsOfInterestInputModels;
     if (DeliveryType.fromFriendlyName(deliveryType).isWebhookDelivery() && StringUtils.isEmpty(webhookUrl)) {
-      throw new IllegalArgumentException(
-                      "Registration is a webhook registration, but missing a webhook url");
+      if (StringUtils.isNotEmpty(runtimeAction)) {
```

Comment:
Can we move this condition inside the enclosing `if` by adding `&& StringUtils.isEmpty(runtimeAction)`?